.azure/tutorials-linux.yml

```diff
@@ -20,48 +18,38 @@ jobs:
 
       - bash: |
           set -e
-          git clone https://github.com/Qiskit/qiskit-tutorials --depth=1
           python -m pip install --upgrade pip setuptools wheel
-          python -m pip install -U \
-            -c constraints.txt \
-            -r requirements.txt \
-            -r requirements-dev.txt \
-            "qiskit-ibmq-provider" \
-            "qiskit-aer" \
-            "z3-solver" \
-            "networkx" \
-            "matplotlib>=3.3.0" \
-            sphinx \
-            nbsphinx \
-            sphinx_rtd_theme \
-            "tweedledum ; python_version < '3.11'" \
-            cvxpy \
-            -e .
+          python -m pip install -U "tox<4.4.0"
           sudo apt-get update
           sudo apt-get install -y graphviz pandoc
-          pip check
+
+          git clone https://github.com/Qiskit/qiskit-tutorials --depth=1
+          mv qiskit-tutorials/tutorials/algorithms/** docs/tutorials/algorithms
+          mv qiskit-tutorials/tutorials/circuits/** docs/tutorials/circuits
+          mv qiskit-tutorials/tutorials/circuits_advanced/** docs/tutorials/circuits_advanced
+          mv qiskit-tutorials/tutorials/operators/** docs/tutorials/operators
         displayName: 'Install dependencies'
-        env:
-          SETUPTOOLS_ENABLE_FEATURES: "legacy-editable"
 
       - bash: |
-          set -e
-          cd qiskit-tutorials
-          sphinx-build -b html . _build/html
-        env:
-          QISKIT_PARALLEL: False
+          tox -e tutorials
+          mkdir -p updated-tutorials/{algorithms,circuits,circuits_advanced,operators}
+          mv docs/tutorials/algorithms/*.nbconvert.ipynb updated-tutorials/algorithms/
+          mv docs/tutorials/circuits/*.nbconvert.ipynb updated-tutorials/circuits/
+          mv docs/tutorials/circuits_advanced/*.nbconvert.ipynb updated-tutorials/circuits_advanced/
+          mv docs/tutorials/operators/*.nbconvert.ipynb updated-tutorials/operators/
```

Comment:
We probably want to maintain `set -e`, and likely also add `shopt -s failglob` here.

Reply:
... unless the idea was that on a failed `tox` run we'd still attempt to produce a partial build artifact?

Reply:
In the big refactor I did of all this to make it more resilient, the upshot is that a partial success will now upload the partial artifacts, whereas before the upload job couldn't have triggered because of the implicit `condition: succeeded()`.